### PR TITLE
Fix Filebeat version reference

### PR DESCRIPTION
**Pull request by plirglo**

### CODE REVIEW

**Comment by przemyslavic** [comment]
Fix for the issue introduced in #2219 (that has been merged but not tested yet).

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-system-service.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@
   vars:
     _packages:
       Debian:
-        - filebeat={{ filebeat_version }}
+        - filebeat={{ specification.filebeat_version }}
```
Comment (atsikham) [inline core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-system-service.yml:11]: Why it's taken from specification if we have that specified in the role's defaults?

**Review by atsikham — CHANGES_REQUESTED** [review]

**Comment by atsikham** [comment]
We also have #2238 PR, what do you think about combining them both?

**Comment by przemyslavic** [comment]
We have it in 2 places 😱 
https://github.com/epiphany-platform/epiphany/blob/v0.8/core/src/epicli/data/common/defaults/configuration/filebeat.yml

https://github.com/epiphany-platform/epiphany/blob/v0.8/core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml

Looks like something has not been properly backported.

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-system-service.yml (hunk 1 above)
Comment (plirglo) [reply core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-system-service.yml:11]: Yes, it looks like: https://github.com/epiphany-platform/epiphany/blob/v0.8/core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml

**Comment by atsikham** [comment]
I see that this refactoring was done in #1793. Imo the right way is to remove this property from `defaults/configuration/filebeat.yml` as it's done in latest versions, not to allow user specify it. Then rename it in role's defaults, `specification` shouldn't be there.

**Comment by rafzei** [comment]
That's right. The role was changed in v0.9.0. Changes wasn't backported so I think we shouldn't merge this PR, instead do a backport.

**Comment by plirglo** [comment]
Maybe we should just fix existing bug to have working configuration (since it's already done in this PR) and create new issue with backporting Filebeat upgrade and role refactoring?

**Review by rafzei — APPROVED** [review]
@plirglo  please create a new issue with backporting missing changes to 0.8

**Review by atsikham — APPROVED** [review]

**Comment by plirglo** [comment]
Created issue #2260

**Pull request merged**